Codeinwp/optimole-wp
# fix: external link on a new line

### All Submissions:

 

### Changes proposed in this Pull Request:
Make sure the external link in Extra badge option appears full instead of icon being extended to the second line.
 

Closes #634.

### How to test the changes in this Pull Request:

1. Make sure the external link in Extra badge option appears full instead of icon being extended to the second line.
2. If you resize the screen to smaller display, it will appear on the same line if there is enough space.

### Other information:
